sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry Azure Monitor Exporter
```

Comment:
nit: consider keeping the style consistent with other exporters? https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/src/OpenTelemetry.Exporter.Prometheus